src/comments.js

```diff
@@ -425,6 +445,35 @@ function printComment(commentPath) {
   }
 }
 
+function findExpressionIndexForComment (expressions, comment, options) {
```

Comment:
@vjeux Then this helper function is invoked in order to detect the correct expression's index. Flow and Babylon don't handle it the same way, that's why it's a bit redundant (but still different).